julia/core.py

```diff
@@ -193,6 +193,15 @@ def module_functions(julia, module):
             pass
     return bases
 
+def determine_if_statically_linked():
+    """Determines if this python executable is statically linked"""
+    # Windows and OS X are generally always dynamically linked
+    if not sys.platform.startswith('linux'):
+        return False
+    lddoutput = subprocess.check_output(["ldd",sys.executable])
+    return not lddoutput.conains("libpython")
```

Comment:
Yes already fixed locally. Give me a few more mins and I'll push all the windows code.